crypto/arm_arch.h

```diff
@@ -82,6 +82,9 @@ extern unsigned int OPENSSL_armv8_rsa_neonized;
 # define ARMV8_SM3       (1<<9)
 # define ARMV8_SM4       (1<<10)
 
+# define ARMV8_SHA3      (1<<10)
+# define ARMV8_UNROLL8_EOR3      (1<<11)
```

Comment:
oops, this collides now with ARMV8_SM4 above.
please use the next free bit.

Reply:
Got it, push it in another machine and at the same time, the PR for sm4 be merged, so the conflict happen, will fix it in next commit.

Reply:
maybe remove the empty line between your items and the old ones too, so git findes a merge conflict when
that happens again...

Reply:
Done.

Reply:
@xkqian it seems you forget to force-push yours last changes.

Reply:
> @xkqian it seems you forget to force-push yours last changes.

Thanks for the reminder. After pass the internal review,  I will push it today.